eduvpn/ui/stats.py

```diff
@@ -0,0 +1,138 @@
+import os
+import logging
+import time
+from datetime import timedelta
+
+from ..nm import get_iface, get_ipv4, get_ipv6, get_timestamp
+
+logger = logging.getLogger(__name__)
+
+LINUX_STATS_FILE = "/sys/class/net/{iface}/statistics/{filename}"
+
+
+class NetworkStats():
+    def __init__(self, interval):
+        self.iface = None
+        self.interval = interval
+        self.addresses = {}
+        self.files = {}
+        self.start_bytes = {}
+        self.timestamp = None
+        self.default_text = "N/A"
+
+    def open_file(self, filename):
+        """
+        Helper function to open a statistics network file
+        """
+        filepath = LINUX_STATS_FILE.format(iface=self.iface, filename=filename)
+        if not os.path.isfile(filepath):
+            logger.info(f"Network Stats: {filepath} is not a file")
+            return None
+        return open(filepath, "r")
+
+    def get_human_readable_bytes(self, total_bytes):
```

Comment:
we can move this method to utils, no need to find it to this class

Reply:
yep makes sense!